### PR TITLE
List mfa devices

### PR DESCRIPTION
**Pull request by frms-**
Hi,
This implements the IAM ListMFADevices command.
/Fredrik

### CODE REVIEW

File: Aws/Iam/Commands/ListMFADevices.hs [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import Text.XML.Cursor (laxElement, ($//), (&|))
+-- | Lists the MFA devices. If the request includes the user name,
+-- then this action lists all the MFA devices associated with the
+-- specified user name. If you do not specify a user name, IAM
+-- determines the user name implicitly based on the AWS access key ID
+-- signing the request.
+--
+-- <https://docs.aws.amazon.com/IAM/latest/APIReference/API_ListMFADevices.html>
+
+data ListMFADevices = ListMFADevices
```
Comment (aristidb) [inline Aws/Iam/Commands/ListMFADevices.hs]: I'd prefer ListMfaDevices for consistency. The same goes for the module name.

File: Aws/Iam/Commands/ListMFADevices.hs [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+instance SignQuery ListMFADevices where
+  type ServiceConfiguration ListMFADevices = IamConfiguration
+  signQuery ListMFADevices{..} = iamAction' "ListMFADevices"
+                                 ([ ("UserName",) <$> lmfaUserName ]
+                                 <> markedIter lmfaMarker lmfaMaxItems)
+
+data ListMFADevicesResponse = ListMFADevicesResponse
+                              { lmfarMFADevices :: [MFADevice]
+                                -- ^^List of 'MFA Device's.
```
Comment (aristidb) [inline Aws/Iam/Commands/ListMFADevices.hs]: Are the two ^ deliberate?

**Comment by aristidb** [comment]
Looks good to me, other than some minor nitpickings.

File: Aws/Iam/Commands/ListMFADevices.hs (hunk 2 above)
Comment (frms-) [inline Aws/Iam/Commands/ListMFADevices.hs]: nope

File: Aws/Iam/Commands/ListMFADevices.hs (hunk 1 above)
Comment (frms-) [inline Aws/Iam/Commands/ListMFADevices.hs]: ok

**Pull request merged**